<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="logout_button_parameters">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1347</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_length">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input Train Length...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_mass">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input Train Mass...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_height">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input Train Height...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_pass">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input Train Pass Count...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_width">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input Train Width...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_crew">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input Train Crew Count...</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>641</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>203</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Train #1 Parameters</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_button">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>280</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1347</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(182, 215, 168);</string>
     </property>
     <property name="text">
      <string>Save Parameters</string>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_ki">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Input Train KI...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="in_kp">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="placeholderText">
      <string>Input Train KP...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pagePtoM_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;- Menu</string>
     </property>
    </widget>
    <widget class="QLabel" name="save_alert">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(221, 221, 221);</string>
     </property>
     <property name="text">
      <string>Saved successfully</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>save_button</zorder>
    <zorder>in_kp</zorder>
    <zorder>in_ki</zorder>
    <zorder>pagePtoM_button</zorder>
    <zorder>save_alert</zorder>
   </widget>
   <zorder>frame</zorder>
   <zorder>logout_button_parameters</zorder>
   <zorder>in_length</zorder>
   <zorder>in_mass</zorder>
   <zorder>in_height</zorder>
   <zorder>in_pass</zorder>
   <zorder>in_width</zorder>
   <zorder>in_crew</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
